plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -147,6 +147,12 @@
     private final ZoneOffset fixedOffsetEast = ZoneOffset.ofHoursMinutes(2, 17);
     private final ZoneOffset fixedOffsetWest = ZoneOffset.ofHoursMinutes(-7, -31);
 
+    private final ZoneId zoneIdGmt = ZoneId.of("GMT");
+    private final ZoneId zoneIdUtc = ZoneId.of("UTC");
+    @SuppressWarnings("ZoneIdOfZ") // there's an equivalent static field, but we'd like to test it anyway
+    private final ZoneId zoneIdZ = ZoneId.of("Z");
```

Comment:
Is it equivalent or "mostly equivalent"?
It surely describes the same zone, but is it indiscernible?
For example GMT, UTC and Z are _not_  completely the same thing.

Reply:
I wasn't actually sure, perhaps I'm too trusting in what ErrorProne says :) So I ran this:

```
jshell 
|  Welcome to JShell -- Version 11.0.10
|  For an introduction type: /help intro

jshell> var z = java.time.ZoneId.of("Z");
z ==> Z

jshell> var utc = java.time.ZoneOffset.UTC;
utc ==> Z

jshell> z == utc;
$3 ==> true

jshell> 
```

So it looks like they are exactly the same, at least in the JDK version I tried.